oap-server/server-configuration/configuration-nacos/src/main/java/org/apache/skywalking/oap/server/configuration/nacos/NacosConfigWatcherRegister.java

```diff
@@ -92,8 +91,35 @@ public NacosConfigWatcherRegister(NacosServerSettings settings) throws NacosExce
 
     @Override
     public Optional<GroupConfigTable> readGroupConfig(final Set<String> keys) {
-        // TODO: implement readGroupConfig
-        return Optional.empty();
+        Set<String> keys1 = new HashSet<>();
+        keys1.add("test-module.default.testKeyGroup");
+
+        GroupConfigTable groupConfigTable = new GroupConfigTable();
+        keys1.forEach(key -> {
```

Comment:
`keys1` seems never getting initialized properly, and real `keys` are never used. 
I assume you submitted the local testing codes.

Reply:
Even in this case, the UTs are passed. I think UTs miss some important cases.

Reply:
because the `key` is the same as the UTs :(